canonical-web-and-design/ubuntu.com
# Update sitemap.xml for static pages

## Done

- Update sitemap.xml for static pages
## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/static/files/sitemap.xml
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that it looks current
